test/database/ConnectionManagerTest.java

```diff
@@ -0,0 +1,22 @@
+package database;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.junit.jupiter.api.Test;
+
+import config.Setting;
+import junit.framework.Assert;
+
+public class ConnectionManagerTest {
+	
+	private ConnectionManager connectionManager = new ConnectionManager();
+
+	@Test
+	public void testConnectionManager() throws SQLException {
```

Comment:
Method should be named `testGetConnection()`.

Reply:
It is unsolved because in some comments before the class SettingTest was achieving the same porpose as ConnectionManagerTest, for these reason I decided to eliminate this class.